app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -131,6 +129,17 @@ class VariantManagerTest {
         assertTrue(variant.hasFeature(ConceptTest))
     }
 
+    @Test
+    fun verifyNoDuplicateVariantNames() {
```

Comment:
Added this guard to protect against anyone trying to reuse `ml` as an experimental variant